app/scripts/lib/channels/inter-tab.js

```diff
@@ -4,51 +4,41 @@
 
 /**
  * This is a special channel that communicates between two
- * tabs of the same browser. It uses one of two adapaters to
- * communicate: BroadcastChannelAdapter or LocalStorageAdapter.
+ * tabs of the same browser, using BroadcastChannels
+ * to communicate.
  *
- * The preferred adapter is the BroadcastChannelAdapter. It uses
- * the BroadcastChannel API to communicate between browser tabs. See
- * https://developer.mozilla.org/docs/Web/API/Broadcast_Channel_API.
- * The BroadcastChannelAdapter is the prefered way to communicate
- * because it does not write any sensitive data to disk.
+ * See https://developer.mozilla.org/docs/Web/API/Broadcast_Channel_API.
  *
- * LocalStorageAdapter uses localStorage and storage events to communicate
- * between tabs. LocalStorageAdapter is used for legacy browsers without
- * BroadcastChannel API support. Any data sent over this channel
- * can be written to disk, so care must be taken to ensure all
- * sensitive data is erased as soon as it is consumed by the receiving tab.
+ * If BroadcastChannel is not supported, no inter-tab communication occurs.
```

Comment:
nit: maybe add a note about supported Firefox versions here or a link to this PR?